### ISSUE TITLE
Table referencing Enum DataType fails to insert

### ISSUE THREAD

**Issue by jklingsporn** [eclipse-vertx/vertx-sql-client#191]
Version: 0.10.6

I've created a table with several field, including a column referencing an enum DataType. When inserting a value into this table using the `PgClient.preparedQuery(String sql, Tuple arguments, Handler<AsyncResult<PgRowSet>> handler)` method an exception is raised:
```
11:07:30.297 [vert.x-eventloop-thread-0] WARN i.r.pgclient.impl.codec.DataType - Postgres type OID=221038 not handled - using unknown type instead 
io.vertx.core.impl.NoStackTraceThrowable: Values [my_string, 3558751944096947426, 1342, -2033217542, 0.935384979593849, BAZ, {"key":"value"}, [1,2,3], 2018-10-10T11:07:29.338] cannot be coerced to [String, Number, Number, Number, Number, String, String, String, LocalDateTime]
```
During debugging I figured out that the enum could not be mapped to a valid DataType and thus created a warning and later on threw an exception.

The workaround to this is issue is invoking `toString()` or `name()` before adding it to the tuple.

**Comment by jklingsporn** [comment]
Another observation: inserting `null` for columns seems to fail as well.

**Comment by vietj** [comment]
can you provide a reproducer ?

**Comment by jklingsporn** [comment]
https://github.com/jklingsporn/reactive-pg-client/blob/master/src/test/java/io/reactiverse/pgclient/PreparedStatementEnumDataTypeTest.java

**Comment by vietj** [comment]
thanks

**Comment by jklingsporn** [comment]
Okay then. But what about the nullability? How can I set column values to `null`?

**Comment by ybonnel** [comment]
null for enum is fixed by #211

**Comment by tsegismont** [comment]
Closed in #211
